azurelinuxagent/common/conf.py

```diff
@@ -680,3 +681,11 @@ def get_firewall_rules_log_period(conf=__conf__):
     NOTE: This option is experimental and may be removed in later versions of the Agent.
     """
     return conf.get_int("Debug.FirewallRulesLogPeriod", 86400)
+
+
+def get_enable_cgroup_v2_resource_limiting(conf=__conf__):
```

Comment:
Cgroup v2 log collector will be disabled by default. This conf option can be used to opt-in